app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/image_import_workflow/import_dispatcher.rb

```diff
@@ -0,0 +1,40 @@
+class ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::ImageImportWorkflow::ImportDispatcher < ::Job::Dispatcher
+
+  def dispatch
+    pending, = Benchmark.realtime_block(:pending_import_jobs) { pending_import_jobs }
+    running, = Benchmark.realtime_block(:running_import_jobs) { running_import_jobs }
+    busy_src_ids = running.map { |_, dst| dst.map { |job| job[:options][:src_provider_id] } }.flatten
```

Comment:
Minor, but `:options` is a column so you can do `job.options[:src_provider_id]`

Reply:
`.map {}.flatten` can be just `running.flat_map`

Reply:
You might want to `uniq` this set so:
`busy_src_ids = running.flat_map { |_, job| job.options[:src_provider_id] }.uniq`